rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -485,6 +485,17 @@
             ),
             false
     ),
+    RECEIVE_HEADER(
+            CallTransaction.Function.fromSignature(
+                    "receiveHeader",
+                    new String[]{"bytes"},
+                    new String[]{"int256"}
+            ),
+            fixedCost(22_000L),  // TODO: calculate gas cost.
+            (BridgeMethodExecutorTyped) Bridge::receiveHeader,
```

Comment:
Is that cast needed?

Reply:
Probably not. It has all the functions that are not void and it is a good way in view of being able to differentiate it. Anyway I can review it.